gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/ManagedController.java

```diff
@@ -45,4 +46,16 @@ Resource upsertResource(@RequestBody Resource resource) {
   Resource deleteResource(@PathVariable("name") String name) {
     return keelService.deleteResource(name);
   }
+
+  @ApiOperation(value = "Check if an application has managed resources", response = Boolean.class)
+  @RequestMapping(value = "/application/{application}", method = RequestMethod.GET)
+  Boolean hasManagedResources(@PathVariable("application") String application) {
```

Comment:
This strikes me as a weird external API and not likely a pattern we'll want to standardize on.

Reply:
FYI @robzienert

Reply:
Please help me make a better API @ajordens

Reply:
🚔

I think there are a handful of things in here that don't quite align with the Google API Design Guide (unfortunately it's quite a read right now).

In an extreme nutshell, avoid primitive return types and favor plural collections. I'd probably expect an `Application` object that had an attribute indicating whether or not it had managed resources.